apache/nifi-fds
# [NIFI-9554] update node/npm versions and frontend deps

To test:

Please install and use node v16.13.2. In the root of the projext run `npm run ci && npm run watch` and verify the available UX components are available and that the styles match nifi-fds v0.2.0.

NOTE: There will be some warnings displayed in the terminal and in the browser and in the browser console. Those are webpack dev server telling us that some of our deps are not providing licenses. I couldn't find a good way to silence those but they can be ignored as they will only show up in dev mode. Simply close the warning overlay and you should be able to verify the UX/UI components.

Next, run `npm run clean:install` and verify tests pass.

Thank you for submitting a contribution to Apache NiFi Flow Design System.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
     in the commit message?

- [ ] Does your PR title start with either NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.



### For code changes:
- [ ] Have you ensured that a full build and that the full suite of unit tests is executed via npm run clean:install at the root nifi-fds folder?
- [ ] Have you written or updated the Apache NiFi Flow Design System demo application to demonstrate any new functionality, provide examples of usage, and to verify your changes via npm start at the nifi-fds/target folder?
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-fds?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-fds?

### For documentation related changes:

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
